ipn/ipnlocal/local.go

```diff
@@ -1868,6 +1868,14 @@ func (b *LocalBackend) Start(opts ipn.Options) error {
 		opts.AuthKey = v
 	}
 
+	if b.state != ipn.Running && b.conf == nil && opts.AuthKey == "" {
+		sysak, _ := syspolicy.GetString(syspolicy.AuthKey, "")
+		if sysak != "" {
+			b.logf("Start: setting opts.AuthKey via syspolicy, len=%v", len(sysak))
```

Comment:
why the length, ooc?

Reply:
It's so that we can debug whether we're getting the right value after changing it within an MDM solution.

Reply:
But aren't they always the same length?

Reply:
ah I see, I didn't know that! That said, fine with leaving it in? It might help support troubleshoot things if folks provide an invalid value via MDM.

Reply:
sure